features/default_app.feature

```diff
@@ -2,7 +2,7 @@ Feature: Plugin integrated in default app
 
 Scenario: Single-module default app builds successfully
     When I build "default_app" using the "standard" bugsnag config
-    And I wait to receive 2 requests
+    And I wait to receive 2 builds
```

Comment:
I'm not convinced that this is the right abstraction because there are two separate requests types here: requests sent to the Build API and requests sent to the Upload API. Is there any possibility that these could be split out in mazerunner, assuming it wouldn't take massive amounts of effort?

The important thing is getting everything on v5 so if it does require lots of effort then it'd probably be ok to skip this

Reply:
It's not a lot of effort - I'll add another "uploads" endpoint to MazeRunner now, which should also be of use for other tools in time.

Reply:
Addition of the new endpoint in Maze Runner is now in development and I'm updating the BAGP tests against the development branch.  Once it's all lined up, I'll create a new PR into this one that updates all the affected scenarios (there are a lot!).